### PR TITLE
Setup release-plan

### PR DESCRIPTION
**Pull request by SergeAstapov**
created using https://github.com/embroider-build/create-release-plan-setup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,14 +25,20 @@
   "prettier": {
     "singleQuote": true
   },
+  "dependencies": {},
   "devDependencies": {
     "@release-it-plugins/lerna-changelog": "^5.0.0",
     "@release-it-plugins/workspaces": "^3.2.0",
-    "release-it": "^15.5.0"
+    "release-it": "^15.5.0",
```
Comment (void-mAlex) [inline package.json:32]: do we still require release it?

File: package.json [hunk 2]
```diff
@@ -25,14 +25,20 @@
...
+    "release-plan": "^0.8.0"
   },
   "engines": {
     "node": "16.* || 18.* || >= 20"
   },
+  "volta": {
+    "node": "16.20.2",
+    "yarn": "1.22.17"
```
Comment (void-mAlex) [inline package.json:40]: this seems wrong since we switched to pnpm

File: package.json (hunk 1 above)
Comment (SergeAstapov) [reply package.json]: nope, thanks for noticing. removed!

File: package.json (hunk 2 above)
Comment (SergeAstapov) [reply package.json]: indeed! removed

**Review by villander — CHANGES_REQUESTED** [review]
I don't think we should switch our release workflow. it has been working well and I don't see any advantage using the release plan over what we have in place

**Comment by SergeAstapov** [comment]
@villander `release-plan` does bit more automation compared to `release-it`, e.g. release preview PR so worth the shot (if we can make it work reliably easily)

**Comment by void-mAlex** [comment]
> I don't think we should switch our release workflow. it has been working well and I don't see any advantage in using the release plan over what we have in place @SergeAstapov @void-mAlex

the issue with the current release flow is that it's not possible, due to mainly no npm access to the package

the change to release-plan ensures that npm access can be given only as a GH secret and access managed on GH with correct perissions to merge, it also ensures the builds are correct and reproducible with no human error 'possible' in the process as the only thing it involves are correct pr titles and labels
at the same time we get more automation and control over the release and if there is something else we need release plan to do it's easier to upstream as long as we have the use case.

for those reasons but mainly the ability to centrally release are why we need this before we merge the rest

**Comment by villander** [comment]
thanks for the context @void-mAlex that makes sense now. We can move forward @SergeAstapov given the release has been a pain point to us requiring Chris to publish npm package every single time

**Review by villander — APPROVED** [review]

**Comment by SergeAstapov** [comment]
Thank you @void-mAlex @mansona !

**Pull request merged**